### ISSUE TITLE
recording::compute_spikecount: peak_time is not computed with the final auto_threshold value

### ISSUE THREAD

**Issue by anilbey** [BlueBrain/BluePyEfe#130]
In `Recording::compute_spikecount` the threshold value that goes to `self.auto_threshold` is modified with the `numpy.clip` command after it is read for extracting the `peak_time` feature. Shouldn't the same threshold value used in `peak_time` be stored in `self.auto_threshold` (to be used in computing other features later)?

```python
            tmp_settings["Threshold"] = thresh
            efel_vals = self.call_efel(['peak_time'], tmp_settings)
            self.peak_time = efel_vals[0]['peak_time']
            # The threshold cannot be lower than the base voltage (handles the case
            # where the step is hyperpolarizing)
            self.auto_threshold = numpy.clip(thresh, base_voltage + offset_voltage, 50.)
```

https://github.com/BlueBrain/BluePyEfe/blob/master/bluepyefe/recording.py#L306:L311

**Comment by DrTaDa** [comment]
Yeah, that would be much better indeed.

**Comment by anilbey** [comment]
Implemented in #131
